stdlib/public/Concurrency/TaskSleep.swift

```diff
@@ -12,6 +12,13 @@
 import Swift
 @_implementationOnly import _SwiftConcurrencyShims
 
+@usableFromInline
+enum swift_clock_id: Int32 {
```

Comment:
- `@usableFromInline` declarations are ABI, so they must come with an availability declaration.
- Please follow the Swift naming conventions even for internal names. (And especially when they are ABI.)

```suggestion
@available(SwiftStdlib 9999, *)
@usableFromInline
enum _ClockID: Int32 {
```